packages/contracts/src/contracts/current/test/TestLibBytes/TestLibBytes.sol

```diff
@@ -25,6 +25,23 @@ contract TestLibBytes is
     LibBytes
 {
 
+    /// @dev Performs a deep copy of a section of a byte array.
+    /// @param b Byte array that will be sliced.
+    /// @param startIndex Index of first byte to copy.
+    /// @param endIndex Index of last byte to copy + 1.
+    /// @return A deep copy of b from startIndex to endIndex.
+    function publicDeepCopyBytes(
```

Comment:
Can we perhaps move all functions used solely in tests into a separate section with a comment stating that they are only used in tests?

Reply:
Is the `test` directory not descriptive enough?